packages/concerto-cto/lib/parser.pegjs

```diff
@@ -1711,22 +1712,43 @@ ImportType
   }
 
 ImportTypes
-    = ImportToken __ ns:QualifiedNamespaceDeclaration ".{" _ types:commaSeparatedIdentifiers _ "}" __ u:FromUri? {
+    = ImportToken __ ns:QualifiedNamespaceDeclaration ".{" _ types:commaSeparatedTypes _ "}" __ u:FromUri? {
```

Comment:
It is interesting that we don't allow spaces before the curly braces or the dot. But yeah, no need to break it now I guess.

Reply:
Yes, in this grammar, I have only modified the rule for `commaSeparatedTypes`.